src/sentry/static/sentry/app/views/settings/account/accountSecurity/components/recoveryCodes.jsx

```diff
@@ -34,36 +35,70 @@ class RecoveryCodes extends React.Component {
 
     if (!isEnrolled || !codes) return null;
 
+    let formattedCodes = codes.join('\n');
+
+    let printCodes = () => {
+      let iframe = window.frames.printable;
+      iframe.document.write(codes.join('<br>'));
+      iframe.print();
+      iframe.document.close();
+    };
+
     return (
       <Panel css={{marginTop: 30}}>
         <PanelHeader>
           <Flex align="center">
-            <Box flex="1">{t('Unused Codes')}</Box>
+            <Box ml={2}>{t('Unused Codes')}</Box>
+            <Box ml={3}>
+              <Clipboard hideUnsupported value={formattedCodes}>
```

Comment:
imo these would look better if grouped to the right with "Regenerate Codes", but maybe is a better question for design.